app/assets/stylesheets/_state-file.scss

```diff
@@ -117,14 +117,14 @@
 
     .progress-step {
       height: 10px;
-      background: $color-grey-medium-light;
+      background: #808080;
       flex-shrink: 1;
```

Comment:
did not create a new shared color label because these colors are just specific to this element